dahu/core/app/scripts/templates/views/workspace/actions/move.html

```diff
@@ -1,7 +1,6 @@
-<div class="move">
-    {{type}}
-    {{target}}
-    {{trigger}}
-    {{finalAbs}}
-    {{finalOrd}}
-</div>
\ No newline at end of file
+<li> trX : 
```

Comment:
don't use li.... please read documentation about HTML form.
Use <label for="blabla">Name of label</label>